components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -213,13 +214,14 @@
 <bnd_topo hgrid="4x5"       >atm/cam/topo/USGS-gtopo30_4x5_remap_c050520.nc</bnd_topo>
 <bnd_topo hgrid="10x15"     >atm/cam/topo/USGS-gtopo30_10x15_remap_c050520.nc</bnd_topo>
 
-<bnd_topo hgrid="ne4np4"    >atm/cam/topo/USGS-gtopo30_ne4np4_16x.c20160612.nc</bnd_topo>
-<bnd_topo hgrid="ne11np4"   >atm/cam/topo/USGS-gtopo30_ne11np4_16xconsistentSGH.c20160612.nc</bnd_topo>
-<bnd_topo hgrid="ne16np4"   >atm/cam/topo/USGS-gtopo30_ne16np4_16xconsistentSGH.c20160612.nc</bnd_topo>
-<bnd_topo hgrid="ne30np4"   >atm/cam/topo/USGS-gtopo30_ne30np4_16xdel2-PFC-consistentSGH.nc</bnd_topo>
-<bnd_topo hgrid="ne60np4"   >atm/cam/topo/USGS-gtopo30_ne60np4_16xconsistentSGH.c20140517.nc</bnd_topo>
-<bnd_topo hgrid="ne120np4"  >atm/cam/topo/USGS-gtopo30_ne120np4_16xdel2-PFC-consistentSGH.nc</bnd_topo>
-<bnd_topo hgrid="ne240np4"  >atm/cam/topo/USGS-gtopo30_ne240np4_16xconsistentSGH.c20130724.nc</bnd_topo>
+<bnd_topo hgrid="ne4np4"   npg="0">atm/cam/topo/USGS-gtopo30_ne4np4_16x.c20160612.nc</bnd_topo>
+<bnd_topo hgrid="ne11np4"  npg="0">atm/cam/topo/USGS-gtopo30_ne11np4_16xconsistentSGH.c20160612.nc</bnd_topo>
+<bnd_topo hgrid="ne16np4"  npg="0">atm/cam/topo/USGS-gtopo30_ne16np4_16xconsistentSGH.c20160612.nc</bnd_topo>
+<bnd_topo hgrid="ne30np4"  npg="0">atm/cam/topo/USGS-gtopo30_ne30np4_16xdel2-PFC-consistentSGH.nc</bnd_topo>
+<bnd_topo hgrid="ne30np4"  npg="2">atm/cam/topo/USGS-gtopo30_ne30pg2_16xdel2-PFC-consistentSGH.c20190417.nc</bnd_topo>
```

Comment:
New hor. grids are called, say, ne30np4.pg2 above but here they are defined as hgrid="ne30np4"  npg="2" . Could you explain why there are 2 different naming conventions in CAM?

Reply:
This approach was adapted from Peter Lauritzen's implementation of the physgrid in CAM6. The np4 and pg2 versions of a given grid share certain input files, so it's advantageous to have the dynamics grid and npg value be carried in separate variables so that thte npg value can modified certain values and not others. Additionally, I think there are XML parsing issues  if we were to try and use hgrid="ne30np4.pg"

Reply:
I am asking because both versions are used. You need to keep nag parameter in cime, but is there a reason to have hgrid=ne30np4 in one file and ne30np4.pg2 in another? If so, why?

Reply:
Looking back at the commits, I think this is a place where it wasn't parsing "ne30np4.pg2" correctly, so I changed it to use the npg variable and it worked. I'm not an expert on how XML works, so there might be a better way to do this that would allow us to be consistent.